### ISSUE TITLE
Minimizing dependencies in the go.mod file

### ISSUE THREAD

**Issue by szkiba** [grafana/xk6-dashboard#103]
To experimentally integrate xk6-dashboard into k6, it is important to have as few dependencies as possible. Integration should introduce as few new dependencies as possible.

Currently, xk6-dashboard would introduce the following new dependencies (they are not included in k6's current dependencies):

```
github.com/gorilla/schema
github.com/magefile/mage
github.com/pkg/browser
github.com/princjef/mageutil
github.com/r3labs/sse/v2
```

The following two of these make the development process more comfortable:

```
github.com/magefile/mage
github.com/princjef/mageutil
```

Since these are not built in, they can be removed from the dependencies with proper organization.

The following dependency was added for more convenient handling of parameters, and can be removed relatively easily:

```
github.com/gorilla/schema
```

In summary, the number of dependencies can be reduced to two new dependencies:

```
github.com/pkg/browser
github.com/r3labs/sse/v2
```
